docs/endpoints/post-optout-status.md

```diff
@@ -0,0 +1,122 @@
+---
+title: POST /optout/status
+description: Checks the opt-out status of raw UID2s.
+hide_table_of_contents: false
+sidebar_position: 03
+---
+
+import Link from '@docusaurus/Link';
+
+# POST /optout/status
+
+Checks the opt-out status of raw UID2s. Given a list of raw UID2s, this endpoint returns the raw UID2s that have opted out, as well as the time that the opt-out took place. For more information, see [User Opt-Out](../getting-started/gs-opt-out.md).
+
+Used by: This endpoint is used by advertisers, data providers, DSPs, and sharers. More generally, this is for participants who have access to raw UID2s but do not have access to the underlying email addresses or phone numbers and want to know opt-out status.
+
+For details, refer to the following documentation, depending on your role:
+
+- [Advertiser/Data Provider Integration Guide](../guides/advertiser-dataprovider-guide.md)
+- [DSP Integration Guide](../guides/dsp-guide.md)
+- [UID2 Sharing: Overview](../sharing/sharing-overview)
+
+## Batch Size and Request Parallelization Requirements
+
+The key guidelines for managing batches of requests to this endpoint are as follows:
+
+- To check the opt-out status of a large number of UID2s, send them in sequential batches with a maximum batch size of 5,000 items per batch.
+- Unless you are using a Private Operator, do not send batches in parallel. In other words, use a single HTTP connection and send batches of raw UID2s consecutively.
+
+## Request Format
+
+`POST '{environment}/v2/optout/status'`
+
+:::important
+You must encrypt all requests using your secret. For details, and code examples in different programming languages, see [Encrypting Requests and Decrypting Responses](../getting-started/gs-encryption-decryption.md).
+:::
+
+### Path Parameters
+
+| Path Parameter | Data Type | Attribute | Description |
+| :--- | :--- | :--- | :--- |
+| `{environment}` | string | Required | Integration environment: `https://operator-integ.uidapi.com`.<br/>Production environment: `https://prod.uidapi.com`.<br/>For a full list, including regional operators, see [Environments](../getting-started/gs-environments.md). |
+
+:::note
+The integration environment and the production environment require different <Link href="../ref-info/glossary-uid#gl-api-key">API keys</Link>.
+:::
+
+### Unencrypted JSON Body Parameters
+
+There is a single body parameter.
+
+| Body Parameter | Data Type | Attribute | Description | Format |
```

Comment:
"Format" is an empty column here. Probably better to remove it ?

Reply:
Fixed in follow-up PR. Thanks!